yaas/cluster.py

```diff
@@ -1,4 +1,182 @@
+        help="Cluster creation template")
+    create_args, _ = parser.parse_known_args(args)
+
+    response = requests.post(
+        config.href('/api/v1/clusters/' + create_args.name),
+        data=create_args.template.read(),
+        **config.requests_opts())
+    if config.args.raw:
+        pprint.pprint(response.json())
+        sys.exit(0)
+    response.raise_for_status()
+
+def _show(parser, args):
+    parser.add_argument(
+        'cluster',
+        nargs='+',
+        help="Shut down all services in cluster")
+    show_args, _ = parser.parse_known_args(args)
+    for cluster in show_args.cluster:
+        response = requests.get(
+            config.href('/api/v1/clusters/' + cluster),
+            params={'fields': "alerts,services,hosts"},
+            **config.requests_opts())
+        if config.args.raw:
+            pprint.pprint(response.json())
+            sys.exit(0)
+        response.raise_for_status()
+        res = response.json()
+
+        print("Alerts")
+        for alert_state, num_hosts in res['alerts']['summary'].items():
+            print('\t{0} - {1}'.format(alert_state, num_hosts))
+        print("Services")
+        for service in res['services']:
+            print("\t" + service['ServiceInfo']['service_name'])
+        print("Hosts")
+        for host in res['hosts']:
+            print("\t" + host['Hosts']['host_name'])
+
+def _blueprint(parser, args):
+    parser.add_argument(
+        'cluster',
+        nargs='+',
+        help="Shut down all services in cluster")
+    show_args, _ = parser.parse_known_args(args)
+    for cluster in show_args.cluster:
+        response = requests.get(
+            config.href('/api/v1/clusters/' + cluster),
+            params={'format': "blueprint"},
+            **config.requests_opts())
+        if config.args.raw:
+            pprint.pprint(response.json())
+            sys.exit(0)
+        response.raise_for_status()
+        pprint.pprint(response.json())
+
+def _destroy(parser, args):
+    parser.add_argument(
+        'cluster',
+        nargs='+',
+        help="Remove cluster from ambari (requires cluster to be stopped)")
+    destroy_args, _ = parser.parse_known_args(args)
+    for cluster in destroy_args.cluster:
+        response = requests.delete(
+            config.href('/api/v1/clusters/' + cluster),
+            **config.requests_opts())
+        if config.args.raw:
+            pprint.pprint(response.json())
+            sys.exit(0)
+        response.raise_for_status()
+
+def _start(parser, args):
+    parser.add_argument(
+        'cluster',
+        nargs='+',
+        help="Shut down all services in cluster")
+    destroy_args, _ = parser.parse_known_args(args)
+    for cluster in destroy_args.cluster:
+        service_start_data = json.dumps({
+            "ServiceInfo": {
+                "state": "STARTED",
+            }
+        })
+        response = requests.put(
+            config.href('/api/v1/clusters/' + cluster + '/services'),
```

Comment:
We should discuss giving this is own command: `cluster service` or `cluster services`. We will hit a similar thing with `cluster hosts`.